packages/pagination/PaginationContent.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Util } from 'reactstrap';
+import BlockUI from 'react-block-ui';
+import { usePagination } from './Pagination';
+
+const PaginationContent = ({
+  component: Component,
+  loadingMessage,
+  loading,
+}) => {
+  const { page } = usePagination();
+
+  return (
+    <BlockUI keepInView blocking={loading} message={loadingMessage}>
+      {page.items &&
+        page.items.map((value, key) => {
+          if (!value.key) {
+            // eslint-disable-next-line no-console
+            console.warn(
+              "Warning a Pagination Item doesn't have a key:",
+              value
+            );
+          }
+
+          return <Component key={value.key || key} {...value} />;
```

Comment:
Maybe `...rest` the props and also spread them on here. It would help avoid a component in the middle just to set additional props?